backend/prompt_studio/prompt_studio_core/views.py

```diff
@@ -403,31 +406,41 @@ def fetch_contents_ide(self, request: HttpRequest, pk: Any = None) -> Response:
                 f"{FileViewTypes.SUMMARIZE.lower()}/"
                 f"{filename_without_extension}.txt"
             )
-
-        file_path = file_path = FileManagerHelper.handle_sub_directory_for_tenants(
-            UserSessionUtils.get_organization_id(request),
-            is_create=True,
-            user_id=custom_tool.created_by.user_id,
-            tool_id=str(custom_tool.tool_id),
-        )
-        file_system = LocalStorageFS(settings={"path": file_path})
-        if not file_path.endswith("/"):
-            file_path += "/"
-        file_path += file_name
-        # Temporary Hack for frictionless onboarding as the user id will be empty
-        try:
-            contents = FileManagerHelper.fetch_file_contents(file_system, file_path)
-        except FileNotFound:
+        if check_feature_flag_status(FeatureFlag.REMOTE_FILE_STORAGE):
```

Comment:
shouldn't this be `if not check_feature_flag_status` ?

Reply:
@kirtimanmishrazipstack This was added for testing purposes, made the change to actual value